shared/ScriptUI.d.ts

```diff
@@ -2519,33 +2519,47 @@ interface ControlPropertiesMap {
 	/**
 	 * A Button object has no creation properties.
 	 * But the third argument to the add() method that creates it can be the initial text value.
+	 * @param name A unique name for the control. For a modal dialog, the special name "ok" makes this defaultElement, and the special name "cancel" makes this the cancelElement of the parent dialog.
```

Comment:
what about this param description
`Special name "ok" makes it primary button, and the special name "cancel" makes it cancel button for current dialog.`